src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2161,6 +2162,18 @@ theorem getLsbD_intMin (w : Nat) : (intMin w).getLsbD i = decide (i + 1 = w) :=
 theorem toNat_intMin : (intMin w).toNat = 2 ^ (w - 1) % 2 ^ w := by
   simp [intMin]
 
+@[simp]
+theorem toInt_intMin {w : Nat} :
+    (intMin w).toInt = -((2 ^ (w - 1) % 2 ^ w) : Nat) := by
```

Comment:
I'm confused: why it there a mod on the RHS?

Reply:
To ensure that for `w = 0` the expression evaluates to `0` instead of `1`. There is precedence of this pattern for here:

```lean
theorem toNat_intMin : (intMin w).toNat = 2 ^ (w - 1) % 2 ^ w := by
  simp [intMin]

Reply:
Let's add a comment on both of those so I'm not confused next time I encounter it. :-)